scripts/Client.js

```diff
@@ -39,66 +40,67 @@ function getRow(row) {
     props.ContactOption = values[0][5];
     props.CalleeNote = values[0][6];
     props.ContactDate = "";
+    //console.log((values[0]));
 //*/
     
-  }).getRow(SPREADSHEET_ID, row);
+  }).getRow(SPREADSHEET_ID, row, campaign);
 }
 
 
 
-
-function getRange(row) {
+function getRange(row) { 
   
   var props = app.currentPage.properties;
   var name = props.Name.toUpperCase();
-
+   
   google.script.run.withSuccessHandler(function(result1) {
     google.script.run.withSuccessHandler(function(result2) {
 //      console.log(result1);      
 //      console.log(result2);
       
       CURR_ROW = result1;
-      END_ROW = result2;
+      START_ROW = result2.start;
```

Comment:
can you please explain this line and the next one?

Reply:
Ahh shoot actually line 61 is supposed to say END_ROW = result2.end. Basically I needed a way to pass both the start and end rows from the back end to the front end, and I did that by having the back end return result2 as an object with "start" and "end" as properties so that I could use both. I cloned your prototype so that I could make changes and experiment without affecting the actual app. The start and end rows are to solve the problem of people potentially accessing and calling people not assigned to them.